src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -56,6 +56,7 @@ class SQLiteAdapter extends PdoAdapter
         self::PHINX_TYPE_TEXT => 'text',
         self::PHINX_TYPE_TIME => 'time_text',
         self::PHINX_TYPE_UUID => 'uuid_text',
+        self::PHINX_TYPE_BINARYUUID => 'binary_blob',
```

Comment:
That seems to be the problem/wrong. See https://github.com/cakephp/migrations/issues/660

Reply:
uuid_blob is the correct one it seems